[GTK][WPE] Unreviewed test gardening

dpino opened the pull request:
#### be0e5c80ab9365a83d18a23fd07766dbdb83e4d2
<pre>
[GTK][WPE] Unreviewed test gardening

Remove redundant baselines.

* LayoutTests/platform/glib/accessibility/accessibility-node-memory-management-expected.txt: Removed.
* LayoutTests/platform/glib/crypto/subtle/aes-export-key-malformed-parameters-expected.txt: Removed.
* LayoutTests/platform/glib/fast/text/fixed-pitch-control-characters-expected.png: Removed.
* LayoutTests/platform/glib/fast/text/fixed-pitch-control-characters-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/content-security-policy/connect-src/shared-worker-connect-src-allowed.sub-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-sizing/replaced-fractional-height-from-aspect-ratio-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/audio_loop_seek_to_eos-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/video_loop_base-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mediacapture-insertable-streams/VideoTrackGenerator.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/current-basic-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-navigations-in-multiple-windows-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entry-after-detach-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-methods/sandboxing-navigate-parent-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/navigation-redirect.https_client-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/painting/parsing/fill-computed-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/painting/parsing/stroke-computed-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/webrtc-stats/outbound-rtp.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https-expected.txt: Removed.
* LayoutTests/platform/glib/svg/W3C-SVG-1.1/pservers-grad-17-b-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/filters/filterRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/masking/maskRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternPreserveAspectRatioA-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternRegions-positioned-objects-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/text/textAnchor2-expected.txt: Removed.
* LayoutTests/platform/glib/svg/css/text-gradient-shadow-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/stroke-width-click-expected.txt: Removed.
* LayoutTests/platform/glib/svg/dom/SVGPathSegList-segment-modification-expected.txt: Removed.
* LayoutTests/platform/glib/svg/filters/feComposite-expected.txt: Removed.
* LayoutTests/platform/glib/svg/filters/sourceAlpha-expected.txt: Removed.
* LayoutTests/platform/glib/svg/foreignObject/svg-document-as-direct-child-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/010-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/011-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/links/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/viewbox/preserveAspectRatio/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/repaint/filter-repaint-expected.txt: Removed.
* LayoutTests/platform/glib/svg/repaint/inner-svg-change-viewPort-relative-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/selection-doubleclick-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/tspan-dynamic-positioning-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/css/css-pseudo/marker-hit-testing-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/editing/run/caret-navigation-around-line-break-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/feature-policy/reporting/xr-reporting.https-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/geolocation/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/mathml/relations/css-styling/not-participating-to-parent-layout-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-09-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-09-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-10-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-10-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-11-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-11-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-13-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-13-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-20-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-20-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-21-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-21-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-26-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-26-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-29-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-29-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-39-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-39-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-64-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-64-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-66-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-66-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-67-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-67-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-68-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-68-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-69-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-69-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-70-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-70-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-77-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-77-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/color-prop-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/color-prop-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-06-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-06-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-units-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-units-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-felem-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-felem-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-turb-01-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-turb-01-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-06-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-06-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/linking-a-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/linking-a-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-fill-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-fill-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-marker-02-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-marker-02-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-stroke-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-stroke-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-10-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-10-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-04-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-04-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-05-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-05-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-06-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-06-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-11-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-11-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-14-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-14-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-pattern-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-pattern-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-01-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-01-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-cond-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-cond-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-image-07-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-image-07-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-use-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-use-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/styling-css-04-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/styling-css-04-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-fonts-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-fonts-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-01-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-01-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/batik/text/textLayout-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/batik/text/textOnPathSpaces-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-1-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-1-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-2-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-2-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-3-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-3-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-4-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-4-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/accname/name/comp_host_language_label-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/script-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/script-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/script-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/script-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/worker-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/worker-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-pseudo/marker-hit-testing-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-table-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-td-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/caption-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/colgroup-col-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/selectors/user-valid-user-invalid-multifield-inputs.tentative-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/editing/run/caret-navigation-around-line-break-expected.txt: Modified.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/geolocation/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/canvas/element/fill-and-stroke-styles/2d.pattern.image.nonexistent-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/selectors/pseudo-classes/inrange-outofrange-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/298825@main">https://commits.webkit.org/298825@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c9666df22b9d6f868b32f31eb0a3a1bab3f845b2

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/116788 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/36452 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/27025 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/122864 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/67374 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/37150 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/45053 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/88688 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/43099 "5 failures") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/119737 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/29621 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/104768 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/69158 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/28684 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/22873 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/66529 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/99000 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/23029 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/125998 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/43687 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/32809 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/97357 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/44051 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/100970 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/97155 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/42480 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/20421 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/39685 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18649 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/43573 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/43040 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/46379 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/44745 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/bcbc06730ec540602e929eda3424a58f57f8608e. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49511

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/bbe3cf3771986c221ff6ec88bb651402f027ec4a. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49511

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/c9666df22b9d6f868b32f31eb0a3a1bab3f845b2. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49511

dpino committed:
[GTK][WPE] Unreviewed test gardening

Remove redundant baselines.

* LayoutTests/platform/glib/accessibility/accessibility-node-memory-management-expected.txt: Removed.
* LayoutTests/platform/glib/crypto/subtle/aes-export-key-malformed-parameters-expected.txt: Removed.
* LayoutTests/platform/glib/fast/text/fixed-pitch-control-characters-expected.png: Removed.
* LayoutTests/platform/glib/fast/text/fixed-pitch-control-characters-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/content-security-policy/connect-src/shared-worker-connect-src-allowed.sub-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-sizing/replaced-fractional-height-from-aspect-ratio-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/audio_loop_seek_to_eos-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/video_loop_base-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mediacapture-insertable-streams/VideoTrackGenerator.worker-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/current-basic-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-navigations-in-multiple-windows-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entry-after-detach-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/navigation-api/navigation-methods/sandboxing-navigate-parent-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/service-workers/service-worker/navigation-redirect.https_client-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/painting/parsing/fill-computed-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/painting/parsing/stroke-computed-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/webrtc-stats/outbound-rtp.https-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https-expected.txt: Removed.
* LayoutTests/platform/glib/svg/W3C-SVG-1.1/pservers-grad-17-b-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/filters/filterRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/masking/maskRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternPreserveAspectRatioA-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternRegions-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/paints/patternRegions-positioned-objects-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/text/textAnchor2-expected.txt: Removed.
* LayoutTests/platform/glib/svg/css/text-gradient-shadow-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/stroke-width-click-expected.txt: Removed.
* LayoutTests/platform/glib/svg/dom/SVGPathSegList-segment-modification-expected.txt: Removed.
* LayoutTests/platform/glib/svg/filters/feComposite-expected.txt: Removed.
* LayoutTests/platform/glib/svg/filters/sourceAlpha-expected.txt: Removed.
* LayoutTests/platform/glib/svg/foreignObject/svg-document-as-direct-child-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/010-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/error/011-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/links/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/hixie/viewbox/preserveAspectRatio/002-expected.txt: Removed.
* LayoutTests/platform/glib/svg/repaint/filter-repaint-expected.txt: Removed.
* LayoutTests/platform/glib/svg/repaint/inner-svg-change-viewPort-relative-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/selection-doubleclick-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/tspan-dynamic-positioning-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/css/css-pseudo/marker-hit-testing-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/editing/run/caret-navigation-around-line-break-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/feature-policy/reporting/xr-reporting.https-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/geolocation/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Removed.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/mathml/relations/css-styling/not-participating-to-parent-layout-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-09-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-09-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-10-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-10-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-11-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-11-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-13-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-13-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-20-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-20-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-21-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-21-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-26-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-26-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-29-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-29-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-39-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-39-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-64-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-64-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-66-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-66-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-67-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-67-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-68-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-68-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-69-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-69-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-70-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-70-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-77-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/animate-elem-77-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/color-prop-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/color-prop-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-06-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-trans-06-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-units-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/coords-units-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-felem-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-felem-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-turb-01-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/filters-turb-01-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-06-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/fonts-elem-06-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/interact-order-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/linking-a-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/linking-a-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/masking-path-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-fill-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-fill-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-marker-02-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-marker-02-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-stroke-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/painting-stroke-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-05-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-05-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-10-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/paths-data-10-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-04-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-04-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-05-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-05-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-06-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-06-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-11-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-11-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-14-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-grad-14-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-pattern-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/pservers-pattern-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-01-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-01-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/render-elems-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-cond-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-cond-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-dom-02-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-image-07-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-image-07-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-use-03-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/struct-use-03-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/styling-css-04-f-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/styling-css-04-f-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-fonts-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-fonts-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-04-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-text-04-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-01-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-01-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-02-t-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-ws-02-t-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/batik/text/textLayout-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/batik/text/textOnPathSpaces-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-1-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-1-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-2-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-2-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-3-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-3-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-4-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/select-x-list-4-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/accname/name/comp_host_language_label-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/script-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/script-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/script-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/script-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/worker-src-self/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/content-security-policy/gen/top.meta/worker-src-wildcard/sharedworker-import.http-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-pseudo/marker-hit-testing-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-table-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/baseline-td-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/caption-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/css-tables/tentative/colgroup-col-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/css/selectors/user-valid-user-invalid-multifield-inputs.tentative-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/editing/run/caret-navigation-around-line-break-expected.txt: Modified.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/geolocation/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/canvas/element/fill-and-stroke-styles/2d.pattern.image.nonexistent-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/selectors/pseudo-classes/inrange-outofrange-expected.txt: Removed.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/selection/contenteditable/modifying-selection-with-non-primary-mouse-button.tentative_middle-expected.txt: Removed.

Canonical link: https://commits.webkit.org/298825@main
